ros_ign_interfaces/msg/Contact.msg

```diff
@@ -3,4 +3,4 @@ ros_ign_interfaces/Entity collision2        # Contact collision2
 geometry_msgs/Vector3[] positions           # List of contact position
 geometry_msgs/Vector3[] normals             # List of contact normals
 float64[] depths                            # List of penetration depths
-geometry_msgs/Wrench[] wrenches             # List of forces/torques
+ros_ign_interfaces/JointWrench[] wrenches   # List of joint wrenches including forces/torques
```

Comment:
I'll just note that changing the definition of a message breaks downstream users, so we need to be mindful about it. Since this message isn't used by anything in this repository yet, and this is targeting the `ros2` branch, I think it's a good move though :+1:

Reply:
Thanks for the heads up!